src/styles/typography.css

```
@@ -0,0 +1,69 @@
+    /* headings */
+    --text-h1: 34px;
+    --text-h1--line-height: 1.25;
+
+    --text-h2: 28px;
+    --text-h2--line-height: 1.3;
+
+    --text-h3: 22px;
+    --text-h3--line-height: 1.35;
+
+    --text-h4: 18px;
+    --text-h3--line-height: 1.35;
```

Biome (2.1.2)

[error] Duplicate custom properties can lead to unexpected behavior and may override previous declarations unintentionally.

--text-h3--line-height is already defined here.

Remove or rename the duplicate custom property to ensure consistent styling.

(lint/suspicious/noDuplicateCustomProperties)